extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php

```diff
@@ -143,6 +147,24 @@ protected function getSourceForChildren() {
 		return array_values( $uniqueParams );
 	}
 
+	protected function getSourceLabelForChildren() {
+		/** @var Node $item */
+		$result = [ ];
+		foreach ( $this->getChildNodes() as $item ) {
+			$label = $item->getSourceLabel();
+			if ( is_array( $label ) ) {
+				$result = array_merge( $result, $label );
+			} else {
+				$source = $item->getSource()[0];
+				if ( !empty( $source ) && !empty( $source ) ) {
+					$result[$source] = $label;
+				}
+			}
```

Comment:
This logic could be done in ``getSourceLabel``, that way it's output would be unified.